### ISSUE TITLE
Dropdown: incorrect focus

### ISSUE THREAD

**Issue by adam-26** [Semantic-Org/Semantic-UI-React#2046]
This applies to use in Chrome.

**Testcase**
https://react.semantic-ui.com/modules/dropdown#dropdown-example-open-on-focus

**Steps to Reproduce**
1. Click on the **down arrow** (NOT on the text of the drop-down) of the **second** dropdown
2. Click on the **down arrow** again
3. Press 'shift + tab' to navigate to the first dropdown

**Expected**
The first dropdown should now have the focus

**Result**
The second dropdown retains the focus. 

** FIX **
This is because when clicking on the **down arrow**, its the `<i ../>` element that obtains the focus. 

Applying the style `pointer-events: none;` to the `<i .../>` element resolves the issue, when the down arrow is clicked focus is passed through to the dropdown component as expected.

**Comment by layershifter** [comment]
1. I opened second dropdown by click on arrow.
2. I closed second dropdown by click on arrow again.

Result: `Dropdown` will lost focus and it looks correct to me. Same will ocurred when item was selected.

**Comment by adam-26** [comment]
The bug is incorrect navigation **between** dropdowns using the keyboard. Did you do step 3?

Yes, the second dropdown loses focus as expected, BUT when you 'shift + tab' to navigate to the FIRST dropdown, the first dropdown DOES NOT have the focus.

**Comment by layershifter** [comment]
@adam-26 Seems you're right, vanilla `select` keeps focus after item select or click, [codepen](https://codesandbox.io/s/946yvn91pp). `blur` is done [there](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/modules/Dropdown/Dropdown.js#L1086) and it has a long history, #627. @levithomason can you clarify this?

**Comment by levithomason** [comment]
We should fix this.  Clicking the Dropdown a second time should retain focus.  This is how both a vanilla HTML select and a [SUI core Dropdown](https://semantic-ui.com/modules/dropdown) behave.

The problem in #627 was that when a Dropdown had focus, and the tab was switched away and back again, the refocused browser tab triggered an additional focus even and the Dropdown would suddenly open.  I'm not sure why, but blurring the Dropdown is not the answer.

The SUI core dropdown retains focus after close and does not re-open when switching tabs away and back.  We can obviously achieve the same result here.

**Comment by adam-26** [comment]
Comment to prevent closure of unresolved bug.

**Comment by levithomason** [comment]
@adam-26 are you able to get a PR going for this?

**Comment by adam-26** [comment]
@levithomason, one way to fix this is apply the style:
```css
pointer-events: none;
```
to the `icon` that renders the dropdown arrow. However, the `<Icon>` component does not accept a `style` prop. I couldn't find an existing class from `semantic-ui` that could be easily applied using the `className` prop.

Should I add an additional class to the core `semantic-ui` repo? Add `style` support to `<Icon>`? 

Please advise.

**Comment by brianespinosa** [comment]
@adam-26 we should probably handle this without using CSS to disable pointer events. I also suspect that adding that style to SUI core styles will potentially break some functionality there. We should handle this in our JS within React instead.

**Comment by levithomason** [comment]
I agree with @brianespinosa here, let's solve this in JS in our repo.

One solution is delegating focus to the Dropdown element itself when the icon is clicked.  We have a `handleIconClick()` method in the Dropdown that someone could experiment with.

**Comment by altimas** [comment]
In our app we have a large menu that is just icons to open up these little side menus. We had the issue where focus would persist on closing the side-menu and not allow you to click the menu item again because the focus was still on it. The work-around I came up with was putting

`inputRef={el => (this.inputElement = el)}>`

into the MenuItem and on the onClick function of the side-menu closing icon I run this

`closeDropdown() {
        this.resetLocalState(); //this is self explanatory
        const hackery = findDOMNode(this.inputElement);
        hackery.blur();
    }`

But even though this "fixed" the issue in Chrome, Firefox, and Safari, it still persists in Edge. Not sure if this even helps but thought I would mention it.

**Comment by layershifter** [comment]
Unstale, please :cat
